### PR TITLE
Deploy Compound III on Optimism

### PR DESCRIPTION
**Pull request by dmitriy-bergman-works**
This PR introduces the deployment for USDC and USDT markets on Optimism as well as the proposal to initialize the market there.

Forum thread - https://www.comp.xyz/t/deploy-compound-iii-on-optimism/4975

### CODE REVIEW

File: deployments/optimism/usdc/configuration.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "supplyBase": 0,
+    "borrowKink": 0.9,
+    "borrowSlopeLow": 0.061,
+    "borrowSlopeHigh": 3.2,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "57_870_370_370e0",
+    "baseBorrowSpeed": "57_870_370_370e0",
```
Comment (kevincheng96) [inline deployments/optimism/usdc/configuration.json:24]: These should be 0 on the deployment.

File: deployments/optimism/usdc/configuration.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  },
+  "assets": {
+    "WETH": {
+      "address": "0x4200000000000000000000000000000000000006",
+      "priceFeed": "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+      "decimals": "18",
+      "borrowCF": 0.83,
+      "liquidateCF": 0.9,
+      "liquidationFactor": 0.95,
+      "supplyCap": "1600e18"
```
Comment (kevincheng96) [inline deployments/optimism/usdc/configuration.json:35]: Supply caps should also be 0 in the initial deployment

File: deployments/optimism/usdc/configuration.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+  "baseTokenPriceFeed": "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+  "pauseGuardian": "0x3fFd6c073a4ba24a113B18C8F373569640916A45",
```
Comment (kevincheng96) [inline deployments/optimism/usdc/configuration.json:7]: Just to double check, was multisig created from the [`clone-multisig`](https://github.com/compound-finance/comet/tree/main?tab=readme-ov-file#clone-multisig) script?

File: deployments/optimism/usdt/configuration.json
Comment (kevincheng96) [inline deployments/optimism/usdt/configuration.json:1]: I recommend splitting out the USDT market. These markets should be deployed and initialized via separate proposals.

File: deployments/optimism/usdc/configuration.json (hunk 3 above)
Comment (dmitriy-bergman-works) [reply deployments/optimism/usdc/configuration.json:7]: Yes

Here the transaction that initiated the creation - https://optimistic.etherscan.io/tx/0x1be034f38200ea3d45e81c4b529ec7a782f26e725ff83edc9b75f46f6ed3891e

Here is the result - https://optimistic.etherscan.io/address/0x3fFd6c073a4ba24a113B18C8F373569640916A45#internaltx

File: deployments/optimism/usdc/configuration.json (hunk 2 above)
Comment (dmitriy-bergman-works) [reply deployments/optimism/usdc/configuration.json]: Updated

File: deployments/optimism/usdc/configuration.json (hunk 1 above)
Comment (dmitriy-bergman-works) [reply deployments/optimism/usdc/configuration.json]: Updated

File: deployments/optimism/usdt/configuration.json
Comment (dmitriy-bergman-works) [reply deployments/optimism/usdt/configuration.json:1]: USDC - https://github.com/compound-finance/comet/pull/838

**Comment by kevincheng96** [comment]
Closing now the #842 is merged

**Pull request closed without merging**